esphome/components/bm8563/bm8563.cpp

```diff
@@ -0,0 +1,155 @@
+#include "bm8563.h"
+#include "esphome/core/log.h"
+
+namespace esphome {
+namespace bm8563 {
```

Comment:
```suggestion
namespace esphome::bm8563 {
```

Change header too and fix comments and brackets at the bottom.